docs/contributing.rst

```diff
@@ -42,31 +35,31 @@ Set up your environment
 Making changes
 --------------
 
-Changes to the theme can be compiled and tested with the default Grunt task:
+Changes to the theme can be compiled and tested with webpacke:
 
 .. code:: console
 
-    grunt
+    npm run dev
 
```

Comment:
Ended up going with run dev instead of run start?

Reply:
The script in `package.json` is named "start". We should fix it or the docs.

Reply:
Aye, this might be non-standard node, but jessetan had the same feedback around `npm run start` . I actually added #797 to maybe move towards putting everything in `setup.py` even :upside_down_face: